src/pages/console/tutorial/buildui.mdx

```diff
@@ -29,31 +29,12 @@ Templates are Figma components that leverage the primitives to enhance your UI e
 
 Back in Studio, choose the “Sync from Figma” button to review all the components from your Figma file. 
 
+![Screenshot showing how to sync with Figma](/images/studio/sync-with-figma.png)
+
 Paste in the URL of your Figma file to start syncing. If this is your first time syncing from Figma, you need to grant Amplify Studio access to your Figma account.
 
 ![Screenshot showing where to get Figma URL](/images/studio/figma-link.png)
 
 Either choose **Accept all** or review individual components with the **Reject** and **Accept** buttons.
 
-![Screenshot showing component review when syncing with Figma](/images/studio/sync-review.png)
-
-## Use component in code
-
-Now that your components are synced, navigate to a component and choose **Configure**. To get your React code, choose **Get component code**.
-
-Follow the instructions in the modal. If this is your first time setting up the UI components in your app, make sure to complete **Initial project setup** first to install all dependencies.
-
-![Screenshot showing how to get the component code](/images/studio/get-code.png)
-
-Next, run `amplify pull` to sync all your components into your local code base. A new `ui-components` folder with all the UI components is created.
-
-Import the UI component you want from `./ui-components` and then render it in your React app. For example:
-
-```js
-import { CardB } from './ui-components'
-...
-  return <div>
-    <CardB />
-  </div>
-...
-```
\ No newline at end of file
+![Screenshot showing component review when syncing with Figma](/images/studio/sync-review.png)
```

Comment:
should we link to "next steps" for using UI components in the frontend app?

Reply:
initial setup has been added now and commited